app/admin/golem/jobs/page.tsx

```
@@ -302,86 +338,437 @@ export default function JobsPage() {
+    return [...filteredJobs].sort((a, b) => {
+      switch (sortBy) {
+        case 'date':
+          return new Date(b.scraped_at).getTime() - new Date(a.scraped_at).getTime();
+        case 'score':
+          return (b.match_score || 0) - (a.match_score || 0);
+        case 'company':
+          return (a.company || '').localeCompare(b.company || '');
+        case 'priority':
+        default:
+          const priorityA = statusPriority[a.status as JobStatus] || 99;
+          const priorityB = statusPriority[b.status as JobStatus] || 99;
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)